2020/prep/submitted/make_spaces_main.sh

```diff
@@ -9,7 +9,7 @@ INC_XML=incremental.xml
 SQ_XML=single-query.xml
 
 MAIN_SOLVERS_CSV=$(mktemp)
-csvgrep -i -c 7 -r '^201[89]-.*$' ${SOLVERS_CSV} > ${MAIN_SOLVERS_CSV}
+csvgrep -i -c 7 -r '^201[89]-.*$' ${SOLVERS_CSV} |csvgrep -i -c 7 -r '.*-fixed$' > ${MAIN_SOLVERS_CSV}
```

Comment:
This introduces a complication that we might want to avoid, namely that to use this we have to have `-fixed` as a reserved keyword. I'd rather explicitly give the IDs.

Reply:
This year we were lucky and we followed this naming convention.  Maybe next year we'll need to do something else, but for now I would leave it as it is.  Note that it's the same for the best-of solvers for 2019 (which included also solvers from 2018).  I myself think this is a pretty simple and efficient way of expressing it, but maybe somebody will send a solver called 2020-OpenSMT-fixed next year and then we'll be in trouble :D

Reply:
That unlikely but annoying to debug scenario is exactly what I'd like to avoid. :) Explicitly blacklisting IDs is easier to debug.

Reply:
I understand.  But listing the IDs is more error-prone, and to reduce the risk of human error there would bring us to a similar solution as in here.  I'm committing a version where the year is hard-coded to the script and which in addition prints a warning.  I hope this solves the problem, but please have a look!

Reply:
IMHO it is okay to have hacks like this.  This is in the year-specific directory, so whoever copies it to the next year should at least look into this.  But maybe one can put a short comment describing the naming convention at the beginning.

Reply:
But it should be `-c "Solver Name"` instead of `-c 7` to make it more robust.
Also maybe combine the two csvgrep to  -r '^(20[0-9][0-9]-.*|.*-fixed)$'

Reply:
Fixed as Jochen suggested (note that file name changed meanwhile).